content/en/blog/2021/stateful-set-made-easier/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "StatefulSets Made Easier With Istio 1.10"
```

Comment:
Could you rename the file "statefulsets-made-easier" to match the CamelCase?

Reply:
done!

Reply:
Actually, @craigbox  this will cause lint error, so I'm rolling back the change.  This is probably why all the other blog folders use lower case only.

```
http://localhost:5153/blog/2021/statefulsets-made-easier/
45
  [404] http://localhost:5153/blog/2021/StatefulSets-made-easier/view-zookeeper-from-kiali.png
46
ERROR: Detected 1 broken links. Scanned 4000 links in 48.742 seconds.
```

Reply:
Lower case please - "statefulsets-made-easier" should work.  (I was asking to remove the hyphen, not change the case 🙂)